cypress/support/fragments/organizations/modals/confirmDeleteOrganizationModal.js

```diff
@@ -0,0 +1,30 @@
+import { Button, Modal } from '../../../../../interactors';
+
+const confirmDeleteItemModal = Modal({ id: 'delete-organization-confirmation' });
+const cancelButton = confirmDeleteItemModal.find(Button('Cancel'));
+const deleteButton = confirmDeleteItemModal.find(Button('Delete'));
+
+export default {
+  waitLoading() {
+    cy.expect(confirmDeleteItemModal.exists());
+  },
+
+  isNotDisplayed() {
+    cy.expect(confirmDeleteItemModal.absent());
+  },
+
+  verifyModalView(organizationName) {
+    cy.expect(Modal({ header: `Delete ${organizationName}?` }).exists());
```

Comment:
I recommend to use confirmDeleteItemModal contant to verify header and message values:
```JS
cy.expect(confirmDeleteItemModal.has({header: `Delete ${organizationName}?`}))
cy.expect(confirmDeleteItemModal.has({message: 'Delete organization?'}))
```